### ISSUE TITLE
Xcode 12, Mac Catalyst target

### ISSUE THREAD

**Issue by jcrate** [SwiftUIX/SwiftUIX#139]
When compiling a Mac Catalyst target, I get this error:

```
.../SourcePackages/checkouts/SwiftUIX/Sources/Intramodular/Documents/FileImportLink.swift:7:8: 
UniformTypeIdentifiers is not available when building for Mac Catalyst. 
Consider using `#if !targetEnvironment(macCatalyst)` to conditionally import this framework.
```

Mac Catalyst compiled fine with Xcode 11.

**Comment by vmanot** [comment]
@jcrate please use the 0.0.2 tag for now. Mac Catalyst is currently broken on master because Apple decided to release Xcode 12GM without the Big Sur SDKs.

**Comment by jcrate** [comment]
I tried the 0.0.2 tag before and that didn't compile either, with a few errors like:

```
.../SwiftUIX/Sources/Intramodular/Controls/ActionLabelView.swift:59:50: 
No type named 'Label' in module 'SwiftUI'
```
despite the fact that it's marked `@available(iOS 14.0, OSX 10.16, tvOS 14.0, watchOS 7.0, *)` and my project has a destination of macOS 10.15. I guess that's also because of the Xcode 12 GM not shipping with Big Sur SDKs. 

The project does compile and run for iOS 13 with SwiftUIX. 

I only used SwiftUIX for a couple things in this project and it won't need to be released before we get an Xcode with 10.16 SDKs. So now that I know what the problem is I'll just close this issue and re-integrate SwiftUIX once 10.16 SDKs are available.
